src/sugar3/speech.py

```diff
@@ -278,7 +278,12 @@ def make_pipeline(self, command):
             self.stop_sound_device()
             del self._pipeline
 
-        self._pipeline = Gst.parse_launch(command)
+        try:
+            self._pipeline = Gst.parse_launch(command)
+        except:
```

Comment:
@ignaciouy, I was talking about something like:

    except ErrorType:
        ...

You should not use catch all except clauses, as you can catch unrelated errors.